internal/tools/securitypolicy/main.go

```diff
@@ -192,48 +195,43 @@ func createPolicyFromConfig(config Config) (sp.SecurityPolicy, error) {
 			}
 			hash := dmverity.RootHash(tree)
 			hashString := fmt.Sprintf("%x", hash)
-			container.Layers = addLayer(container.Layers, hashString)
+			addLayer(&container.Layers, hashString)
 		}
 
-		container.NumLayers = len(layers)
-
 		// add rules for all known environment variables from the configuration
 		// these are in addition to "other rules" from the policy definition file
 		config, err := img.ConfigFile()
 		if err != nil {
 			return p, err
 		}
 		for _, env := range config.Config.Env {
-			rule := sp.SecurityPolicyEnvironmentVariableRule{
-				Strategy: sp.EnvVarRuleString,
+			rule := securitypolicy.EnvRule{
+				Strategy: securitypolicy.EnvVarRuleString,
 				Rule:     env,
 			}
 
-			container.EnvRules = addEnvRule(container.EnvRules, rule)
+			addEnvRule(&container.EnvRules, rule)
 		}
 
 		// cri adds TERM=xterm for all workload containers. we add to all containers
 		// to prevent any possble erroring
```

Comment:
Btw I found what this was from (can't remember if we ever really got to the bottom of it). Upstream cri only sets this if a tty is asked for. However, our fork ALWAYS sets this if the platform (guest in this case) is going to be Linux https://github.com/kevpar/cri/blob/windows_port/pkg/server/container_create_windows.go#L329-L330

Reply:
We'll wanna keep a close eye on this if we ever try and move towards more of upstreams work which we've been trying to.